pkgs/by-name/no/noita-entangled-worlds/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  makeWrapper,
+  steam-run,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "noita_entangled_worlds";
+  version = "0.32.5";
+
+  src = fetchurl {
+    url = "https://github.com/IntQuant/noita_entangled_worlds/releases/download/v${finalAttrs.version}/noita-proxy-linux.zip";
+    sha256 = "sha256-Qx25tvmq/0NDvDnnqw+8HoQP5uzoMXKsK2PsiVntW+4=";
+  };
+
+  nativeBuildInputs = [
+    unzip
+    makeWrapper
+  ];
+  buildInputs = [
+    steam-run
+  ];
+  unpackPhase = ''
+    mkdir source
+    unzip $src -d source
+  '';
```

```suggestion
  src = fetchzip {
    url = "https://github.com/IntQuant/noita_entangled_worlds/releases/download/v${finalAttrs.version}/noita-proxy-linux.zip";
    sha256 = "sha256-qvcSf82RqPoRVwVVWqMcQNEa03b/asLgBwD5hsbyBoE=";
    stripRoot = false;
  };

  nativeBuildInputs = [
    makeWrapper
  ];
  buildInputs = [
    steam-run
  ];
```

Then delete the `source` prefix from the install phase, and don't forget to fix up the top-of-file set of dependencies by removing `unzip` and replacing `fetchurl` with `fetchzip`.